beacon-chain/blockchain/service.go

```diff
@@ -236,77 +243,28 @@ func (c *ChainService) blockProcessing(done <-chan struct{}) {
 			return
 		// Listen for a newly received incoming block from the sync service.
 		case block := <-c.incomingBlockChan:
-			// 3 steps:
-			// - Compute the active state for the block.
-			// - Compute the crystallized state for the block if cycle transition.
-			// - Store both states and the block into a data structure used for fork choice.
-			//
-			// Another routine will run that will continually compute
-			// the canonical block and states from this data structure using the
-			// fork choice rule.
-			var canProcessBlock bool
-			var canProcessAttestations bool
-			var err error
-			var blockVoteCache map[[32]byte]*types.VoteCache
-
-			h, err := block.Hash()
-			if err != nil {
-				log.Debugf("Could not hash incoming block: %v", err)
-			}
-
-			receivedSlotNumber := block.SlotNumber()
-
-			log.WithField("blockHash", fmt.Sprintf("0x%x", h)).Info("Received full block, processing validity conditions")
-
-			parentExists, err := c.chain.hasBlock(block.ParentHash())
+			// 1. Validate the block
```

Comment:
This is a good place to start the review. It may be easier to just view the new code rather than parsing the diff.